### PR TITLE
REST IN PEACE HORNYGRANNY 4000BC-2017AC

### PR DESCRIPTION
**Pull request by Iamgoofball**
```
<Iamgoofball> @seen hg|work
<Pirou> [2016-12-09 20:06:54] *** Quits: hg|work (~hornygran@E5265AAA.D02AE956.6FAA28BB.IP) (Ping timeout: 240 seconds)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## /tg/station
+## /tg/station server and codebase
```
Comment (PKPenguin321) [inline README.md]: a b s o l u t e

**Comment by optimumtact** [comment]
We will forever play in his universe

**Comment by PKPenguin321** [comment]
https://www.youtube.com/watch?v=qoV06aMCpmk

File: README.md (hunk 1 above)
Comment (PKPenguin321) [inline README.md]: also it's server and project

**Comment by Bawhoppen** [comment]
it's BCE and CE now

**Pull request merged**